### PR TITLE
Update german translation

### PR DESCRIPTION
**Pull request by terenc3**

### CODE REVIEW

**Review by TyrenDe — APPROVED** [review]
Thanks for doing this!

File: MixItUp.Base/Resources.de-DE.resx [hunk 1]
```diff
@@ -638,27 +638,27 @@
     <comment>Manual</comment>
   </data>
   <data name="Event" xml:space="preserve">
-    <value>Event</value>
+    <value>Ereigniss</value>
```
Comment (Cerbion) [inline MixItUp.Base/Resources.de-DE.resx:641]: should be "Ereignis" not "Ereignis**s**"

File: MixItUp.Base/Resources.de-DE.resx [hunk 2]
```diff
@@ -638,27 +638,27 @@
...
     <comment>Manual</comment>
   </data>
   <data name="EventBonuses" xml:space="preserve">
     <value>Event Bonusse</value>
     <comment>Manual</comment>
   </data>
   <data name="EventList" xml:space="preserve">
-    <value>Eventliste</value>
+    <value>Ereignissliste</value>
```
Comment (Cerbion) [inline MixItUp.Base/Resources.de-DE.resx:649]: should be "Ereignisliste" not "Ereignis**s**liste"

Comment (TyrenDe) [reply MixItUp.Base/Resources.de-DE.resx:649]: Please do not use pull request comments to recommend changes to strings. The best place is to either make a PR yourself, or come post in our localization channel on our discord.

**Pull request merged**